src/components/offers-list-loader/offers-list-loader.tsx

```diff
@@ -1,10 +1,11 @@
 import ContentLoader from 'react-content-loader';
 import OfferLoader from '../offer-loader/offer-loader.tsx';
 import {OFFERS_LOADER_COUNT} from '../../const.ts';
+import {nanoid} from 'nanoid';
 
 
 function OffersListLoader () {
-  const offerLoaders = Array.from({length: OFFERS_LOADER_COUNT}, (i: number) => i++);
+  const offerLoaders = Array.from({length: OFFERS_LOADER_COUNT}, () => nanoid());
```

Comment:
Там не обязательно было ставить зависимость
```suggestion
  const offerLoaders = Array.from({length: OFFERS_LOADER_COUNT}, () => crypto.randomUUID());
```